docker_mirror.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/python
 # coding=utf8
 
+from __future__ import print_function
+from __future__ import unicode_literals
```

Comment:
Is this import necessary?

Reply:
A future statement is a directive to the compiler that a particular module should be compiled using syntax or semantics that will be available in a specified future release of Python. The future statement is intended to ease migration to future versions of Python that introduce incompatible changes to the language.
[refer to this](https://docs.python.org/2/reference/simple_stmts.html#future)